lib/services/fetch_basket.rb

```diff
@@ -0,0 +1,16 @@
+module Store
+  class FetchBasket
+    def call
+      BASKET.map do |item|
+      {
+        product: item.fetch_product,
+        id: item.id,
+        quantity: item.quantity,
+        price_with_vat: item.price_with_vat,
+        total_price: item.total_price,
+        total_price_with_vat: item.total_price_with_vat
+      }
```

Comment:
@shadrk If you mean this line `id: item.id` then I use it in delete form. Product id in the column ID it's kind of pointless because when for example I delete one product then the numbers in this column won't be still like for example I have four items in basket and I remove item number 3.